api_app/playbooks_manager/migrations/0045_add_ailtyposquatting_free_to_use.py

```diff
@@ -0,0 +1,49 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+
+from django.db import migrations
+
+
+def migrate(apps, schema_editor):
+    playbook_config = apps.get_model("playbooks_manager", "PlaybookConfig")
+    AnalyzerConfig = apps.get_model("analyzers_manager", "AnalyzerConfig")
+
+    pc = playbook_config.objects.get(name="FREE_TO_USE_ANALYZERS")
+    pc.analyzers.add(AnalyzerConfig.objects.get(name="AILTypoSquatting").id)
+    pc.full_clean()
+    pc.save()
+
+    pc = playbook_config.objects.get(name="Sample_Static_Analysis")
+    pc.analyzers.add(AnalyzerConfig.objects.get(name="AILTypoSquatting").id)
+    pc.full_clean()
+    pc.save()
```

Comment:
this is not a sample analyzer, please do not copy and paste without reviewing

Reply:
anyway, I would not add this analyzer to any playbook for now

Reply:
I'm sorry for the overlook. Thanks for the point out!